cwrc_common.rb

```diff
@@ -72,20 +69,42 @@ def self.download_cwrc_obj(cookie, cwrc_obj, cwrc_file)
     retries = [10, 30, 90, 300, 900]
     http_read_timeout = ENV['CWRC_READ_TIMEOUT'].to_i
     begin
```

Comment:
Old code didn't handle HTTP::Net response code checking. Is this the proper way to prevent writing a file? I've added a VCR test for this case.